docs/docs/css/reference.css

```diff
@@ -1,3 +1,8 @@
+html {
+	scroll-padding-top: 5em;
+    overflow: auto;
```

Comment:
And finally, please indent `overflow: auto`.

Reply:
fixed! Actually, `scroll-padding-top` had extra spacing.